R/mergeData.R

```diff
@@ -38,6 +38,7 @@ mergeData <- function(feat_eng_df, clinical_data, match = "ScrID", id_mapping =
   df_merged <- left_join(feat_eng_df, id_mapping, by = "pid") %>%
     dplyr::relocate(PatID, .after = "pid") %>%
     dplyr::select(-pid, -SubjID, -DataDir)
```

Comment:
Why does line 40 exist? As a result, I got the following error:
![Screenshot 2023-09-28 at 10 51 02 AM](https://github.com/aashen12/VoxelProc/assets/66567109/6128f599-30fc-492b-b077-363f82d6490e)
The same will apply to DataDir if that column does not exist. 
Please fix and delete the comment.

Reply:
I included line 40 because I thought the intention was to get rid of all the other IDs that weren't being used. I can just keep it there instead.